zerver/lib/bulk_create.py

```diff
@@ -111,7 +111,7 @@ def bulk_create_streams(realm: Realm,
             options['history_public_to_subscribers'] = (
                 not options.get("invite_only", False) and not realm.is_zephyr_mirror_realm)
         if name.lower() not in existing_streams:
-            from zerver.lib.actions import render_stream_description
+            from zerver.lib.streams import render_stream_description
```

Comment:
I'm pretty sure we can now move this import to the top of the file.

Reply:
Yeah, sorry I missed it.
Gonna change it now